common/views/components/PageLayout/PageLayout.tsx

```diff
@@ -271,13 +290,18 @@ const PageLayoutComponent: FunctionComponent<Props> = ({
             />
           )}
         {popupDialog.data.isShown && <PopupDialog document={popupDialog} />}
-        <div
+        <main
           id="main"
           className="main"
           role={excludeRoleMain ? undefined : 'main'}
```

Comment:
My reading of [this](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/main#:~:text=The%20HTML%20element%20represents,central%20functionality%20of%20an%20application.) is that `main` element has an implicit role of `main` and can't _not_ be `main` – I imagine this is why we've used a `div` with a `role` instead of a `main` element, for instances where it _isn't_ main.

Reply:
Makes sense, I’ll revert this.